src/tutorials/data/queries.md

```diff
@@ -0,0 +1,365 @@
+
+### Get them all!
+
+When pagination is not required by the client, cozy-client offers the [`queryAll`](https://docs.cozy.io/en/cozy-client/api/cozy-client/#cozyclientqueryallquerydefinition-options-array) method:
+
+```javascript
+const queryDef = client.find("io.cozy.todos")
+const docs = await client.queryAll(queryDef)
+```
+
+Note that it will actually automatically paginate if the total number of documents to return is superior to the `limit` (100 by default): this avoids loading too many documents in memory on the server side.
+However, it is possible to overcome this pagination by passing `limit: null`: in this case, cozy-client will request the [`_all_docs` endpoint](https://docs.couchdb.org/en/stable/api/database/bulk-api.html#db-all-docs) and retrieve all the documents in one query. This endpoint is automatically created by CouchDB and is actually a [view](#views) on the `_id` key.
+
+Thus, doing this will actually query the `_all_docs` endpoint and return all the documents in one request:
+
+```javascript
+const queryDef = client.find("io.cozy.todos")
+const docs = await client.query(queryDef, {limit: null})
+```
+
+⚠️ When querying this endpoint, the response includes the [design docs](https://docs.couchdb.org/en/master/ddocs/index.html), which are the Mango indexes and views definitions.  Those documents are automatically filtered for the paginated queries.
+
+⚠️ This method is faster than the pagination as it avoids to make several server requests. However, if there are many documents to return, `_all_docs` queries can take a lot of time to complete and consume server resources. Hence, you should be cautious when using this route.
+
+
+## Sort in CouchDB
+
+When dealing with various types of data, the ascending sort order is the following:
+
+- null
+- booleans
+- numbers
+- strings: “a” < “A” < ”aa” < “b” …
+- arrays
+- objects
+
+See the [CouchDB documentation](https://docs.couchdb.org/en/2.3.1/ddocs/views/collation.html#collation-specification) for more details on the sort orders.
+
+ℹ️ The sort order is the same for Mango queries and views. 
+
+### Comparison of strings
+
+Comparison of strings is done using ICU which implements the Unicode Collation Algorithm, giving a dictionary sorting of keys. This can give surprising results if you were expecting ASCII ordering. Note that:
+
+- All symbols sort before numbers and letters (even the “high” symbols like tilde, `0x7e`)
+- Differing sequences of letters are compared without regard to case, so `a < aa` but also `A < aa` and `a < AA`
+- Identical sequences of letters are compared with regard to case, with lowercase before uppercase, so `a < A`.
+
+
+## Views
+
+⚠️ In Cozy, we disabled by default the possibility to create views from applications. Consequently, [cozy-client](https://github.com/cozy/cozy-client) does not support views creation, because they are disabled for applications. See the [view performances](./advanced.md#views-performances) section to find out why. 
```

Comment:
```suggestion
⚠️ In Cozy, we disabled by default the possibility to create views from applications. Consequently, [cozy-client](https://github.com/cozy/cozy-client) does not support view creation, because they are disabled for applications. See the [view performances](./advanced.md#views-performances) section to find out why. 
```